src/Tests/UnitTest/TestComponent_Tests.cs

```diff
@@ -845,12 +845,46 @@ public void TestVectorGetMany()
             Assert.True(composableObjects.GetRange(1, 3).SequenceEqual(interfaceSubset));
         }
 
+#if NET5_0
+        [Fact]
+        public void Fast_Abi_Simple()
+        {
+            var simple = new test_component_fast.Simple();
+            Assert.Equal("Method1", simple.Method1());

```

Comment:
especially for this change, which rearranges vtable layouts, it would be great to have validation logic on the test_component_fast side, as well (see my comment on that PR).

Reply:
As per offline discussion, a new issue has been created for this.